Tools/Templates/buildable/FlowCoordinatorTemplate/FlowTemplateCoordinatorParameters.swift

```diff
@@ -0,0 +1,33 @@
+/*
+ Copyright 2021 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+
+/// FlowTemplateCoordinator input parameters
+class FlowTemplateCoordinatorParameters {
```

Comment:
Can it be a struct?
From this PR, it is used only from full swift. Else, there is the bridge presenter.